pyproject.toml

```diff
@@ -136,7 +136,11 @@ test-command = [
 profile = "black"
 
 [tool.mypy]
+disallow_untyped_defs = true
```

Comment:
Tell mypy to check functions, even if the function def does not have annotations.

Without this functions like, will pass the mypy check.;
```python
def foo():
  y: list[float] = 0
  return sum(y)
````